ncm-ssh/src/main/pan/components/ssh/schema.pan

```diff
@@ -117,6 +117,7 @@ type ssh_daemon_type = {
     "options" ? ssh_daemon_options_type
     "comment_options" ? ssh_daemon_options_type
     "sshd_path" ? string
+    "validate_config" ? string with match (SELF, '^(yes|no|optional)$')
```

Comment:
please use a boolean to validate or not. 
and use another boolean to allow to skip the validation if sshd is missing (but if `sshd` is missing ,you cannot validate anyway, so this feels like the boolean means something else) 

but why would one ever choose not to validate? and why would we allow that?
i think one boolean to indicate that the test can be skipped if sshd is missing is enough.
btw, what sort of system is this that there is no sshd available? (maybe not the one that you will run, but can't you use a system sshd to validate the config if the one from the network FS is missing? imho it's better than just skipping the config test)